BraintreeCore/src/main/java/com/braintreepayments/api/core/TLSCertificatePinning.kt

```diff
@@ -5,87 +5,714 @@ import java.io.InputStream
 import java.nio.charset.StandardCharsets
 import javax.net.ssl.SSLException
 
+/**
+ * THIS CODE IS GENERATED BY 'codify_certificates.sh' in Braintree Github Enterprise repo
+ * 'braintree-mobile-certificate-generation'. This script concatenates '.crt' files in the 'certificates'
+ * directory.
+ *
+ * Encapsulates our trusted x509 Certificates for Secure SSL Communication with Braintree's servers
+ * to avoid storing certificate files in an 'NSBundle'.
+ *
+ * Do not update this file directly. If new certificates are needed, update the 'certificates' directory
+ * in 'braintree-mobile-certificate-generation' and re-run the certificate generation script.
+ */
+@Suppress("LargeClass")
```

Comment:
Did you have to manually add this suppress annotation? Maybe we can tweak the detekt rule to exclude this class, so the next time we generate the cert, we don't need to have any manual steps.

Reply:
`@Suppress` was added to the new bash script for the `LargeClass` linter error. You can see the script in this PR https://github.braintreeps.com/braintree/braintree-mobile-certificate-generation/pull/7

The script hasn't been merged yet. If there are changes we should make, we can do so in the script PR and I'll generate an updated file to add to Android BT SDK

Reply:
Also how do we exclude the class?

Reply:
Oh nice! I didn't realize the script was adding the suppress annotation.

You can modify the `detekt-config.yml` file to change the ruleset: https://detekt.dev/docs/introduction/configurations#path-filters--excludes--includes

Reply:
Thanks :) I made the changes in `detekt-config.yml`